packages/@webex/plugin-meetings/src/reachability/clusterReachability.ts

```diff
@@ -389,29 +675,49 @@ export class ClusterReachability extends EventsScope {
 
     // Initialize this.result as saying that nothing is reachable.
     // It will get updated as we go along and successfully gather ICE candidates.
-    this.result.udp = {
-      result: this.numUdpUrls > 0 ? 'unreachable' : 'untested',
-    };
-    this.result.tcp = {
-      result: this.numTcpUrls > 0 ? 'unreachable' : 'untested',
-    };
-    this.result.xtls = {
-      result: this.numXTlsUrls > 0 ? 'unreachable' : 'untested',
-    };
+    this.result.udp.result = this.numUdpUrls > 0 ? 'unreachable' : 'untested';
```

Comment:
why has this initialisation been changed here? this PR is big enough and complicated enough without extra unnecessary changes like this. The previous code was ensuring that latencyInMilliseconds and clientMediaIPs are set to undefined whenever we do start(), the new code doesn't do that. Also, I think I've made this comment about these changes on the previous PR.